benchmark/tls/convertprotocols.js

```diff
@@ -8,14 +8,14 @@ const bench = common.createBenchmark(main, {
 });
 
 function main({ n }) {
-  var i = 0;
   var m = {};
   // First call dominates results
   if (n > 1) {
     tls.convertNPNProtocols(['ABC', 'XYZ123', 'FOO'], m);
     m = {};
   }
   bench.start();
-  for (; i < n; i++) tls.convertNPNProtocols(['ABC', 'XYZ123', 'FOO'], m);
+  for (var i = 0; i < n; i++)
+    tls.convertNPNProtocols(['ABC', 'XYZ123', 'FOO'], m);
```

Comment:
Can we store the array in a variable in addition to this change? The results here are probably dominated by the array creation.